test/integration/targets/apt_key/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 # Test code for the apt_key module.
-# (c) 2017, James Tanner <tanner.jc@gmail.com>
+# (c) 2017, 2021 James Tanner <tanner.jc@gmail.com>
```

Comment:
```suggestion
# (c) 2017 James Tanner <tanner.jc@gmail.com>
```

We decided that we will only keep original year of copyright.

Reply:
OK, thank you! I will fix that!